changelog/@unreleased/pr-5530.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: improvement
+improvement:
+  description: |-
+    Reduces backoff upon receiving an exception from KVS.
```

Comment:
let's say Cassandra here, to be clear that postgres/oracle users aren't affected

Reply:
:+1: